docs/en/operations/settings/settings.md

```diff
@@ -3707,7 +3767,7 @@ Possible values:
 
 Default value: 2.
 
-## compatibility {#compatibility}
+### compatibility {#compatibility}
```

Comment:
This setting and `allow_settings_after_format_in_insert` are not related to `Schema inference settings`